Readme.md

```diff
@@ -0,0 +1,79 @@
+# elm-plots
+elm-plots is a plotting framework built entirely in elm using svgs. It draws inspiration from [d3](https://github.com/mbostock/d3) and [vega](https://github.com/vega/vega). Its goal is to make drawing visuals such as plots and charts easy to do. 
```

Comment:
Unpluralize "svgs" since the "g" stands for "graphics". Add an explicit subject to the second sentence. I recommend "The goal of elm-plots is to..."